apps/portal/src/content/docs/foundations/logos.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Logos
+description: Logo component
+---
+
+The `Logo` component renders brand icons from the `simple-icons` package.
+
+import { DocsPage } from "@/components/docs-page";
+import { Logo, LogoNameMap } from "@harnessio/ui/components";
+
+<DocsPage.ComponentExample
```

Comment:
i think this format of documentation works better for components.
for the foundations, let's render a grid of all available logos. we don't need to show code samples for it.

Reply:
Sounds good, will take it up in a follow up PR.